terraform/modules/github-ci-bootstrap/outputs.tf

```diff
@@ -43,4 +63,21 @@ output "github_repository" {
 output "target_projects" {
   description = "Map of target projects configured for this service account"
   value       = var.target_projects
+}
+
+
+output "write_branch_patterns" {
+  description = "List of branch patterns that are allowed to use the read/write service account"
+  value       = var.write_branch_patterns
+}
+
+# Additional outputs for backward compatibility and clarity
```

Comment:
I wonder if it's worth breaking backwards compatibility here.  I guess my suggestions above to rename things to have the `_rw` suffix might do that as well; I'm not sure what symbols here are exported and what symbols are private.

I think it's to our advantage to make folks explicitly choose if they want read-write or read-only access at any point, and having varnames that embed that in the name make that decision more obvious.  Otherwise, I would just use `terraform_service_account_email` and not even realize there was another email variable I could be using.